<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibraryWindow</class>
 <widget class="QMainWindow" name="LibraryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор библиотек</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Библиотека:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="libraryNameText">
        <property name="text">
         <string>Выберите библиотеку</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="libraryNameLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="parametersTableView"/>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLibrary">
    <property name="title">
     <string>Library</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExportNetList"/>
    </widget>
    <addaction name="actionNewLibrary"/>
    <addaction name="actionRemoveLibrary"/>
    <addaction name="actionOpenLibrary"/>
    <addaction name="menuExport"/>
    <addaction name="actionCloseLibrary"/>
   </widget>
   <widget class="QMenu" name="menuParameter">
    <property name="title">
     <string>Parameter</string>
    </property>
    <addaction name="actionAddParameter"/>
    <addaction name="actionRemoveParameter"/>
   </widget>
   <addaction name="menuLibrary"/>
   <addaction name="menuParameter"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewLibrary">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionRemoveLibrary">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionAddParameter">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionRemoveParameter">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionOpenLibrary">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExportNetList">
   <property name="text">
    <string>NetList</string>
   </property>
  </action>
  <action name="actionCloseLibrary">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
